core/services/pipeline/orm.go

```diff
@@ -127,6 +265,36 @@ func (o *orm) GetAllRuns() ([]Run, error) {
 	return runs, err
 }
 
+func (o *orm) GetUnfinishedRuns(now time.Time, fn func(run Run) error) error {
+	return sorm.Batch(func(offset, limit uint) (count uint, err error) {
```

Comment:
Can we either copy the `Batch` function here, or move it to e.g. utils package?

I think we want to try to avoid pulling in dependencies on store or store/orm into these packages

Reply:
I'll place it in the `postgres` package.